From: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
Date: Tue, 5 Mar 2019 10:17:57 +0100
Subject: Add libresolve.so.2 in the initramfs

busybox in buster depends on libresolve.so.2 so copy it to the
initramfs; Potentially in future fakemachine should move to
busybox-static

Signed-off-by: Sjoerd Simons <sjoerd.simons@collabora.co.uk>
---
 machine.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/machine.go b/machine.go
index 4a9bd83..021ee67 100644
--- a/machine.go
+++ b/machine.go
@@ -457,6 +457,7 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 	if mergedUsrSystem() {
 		prefix = "/usr"
 	}
+	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libresolv.so.2")
 	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libc.so.6")
 	w.CopyFile(prefix + "/bin/busybox")
 
